docs/Containerfile.5.md

```diff
@@ -132,6 +134,23 @@ Current supported mount TYPES are bind, and tmpfs.
 
               · tmpcopyup: Path that is shadowed by the tmpfs mount is recursively copied up to the tmpfs itself.
 
+	Options specific to secret:
+
+              · id: identifier for the secret passed into the `buildah bud --secret` or `podman build --secret`.
+
```

Comment:
This is repeated on line 143/162.  Mentioning the option just once would be sufficient.